admin/orders.php

```diff
@@ -718,27 +718,31 @@ function couponpopupWindow(url) {
                         echo '</i></small></span>';
                       }
                     }
+                    // Mobile phones only
+                    echo '<span class="hidden-sm-up">'; 
```

Comment:
`hidden-sm-up` - Is that official? or accidentally works?  I don't see it in the docs

Reply:
Most official. 
https://v4-alpha.getbootstrap.com/layout/responsive-utilities/

Reply:
But ... we're currently using BS 3, not 4
Plus, the v4-alpha docs on that are quite different from the official v4 docs

Reply:
Two weeks ago when I did this, I could have sworn I saw this in the CSS.  Was it the whiskey?  Maybe.

Reply:
Should be .hidden-sm for BS 3.4.1

Reply:
https://getbootstrap.com/docs/3.4/css/#responsive-utilities

Reply:
No it's the opposite.  `hidden-sm-up` means only show on xs.

Reply:
I think all BS styles work that way: All apply from "specified size, and larger, until a larger-size rule is provided"

Reply:
Nope, not with .hidden-*

Reply:
Looks like `visible-xs-inline` is what's needed here.